cli/commands/project_config.go

```diff
@@ -35,33 +36,48 @@ type ProjectConfig struct {
 
 // Create a new ProjectConfig using configured or default locations
 func NewProjectConfig() *ProjectConfig {
+	readyConfig := &ProjectConfig{}
 	projectConfigFile := os.Getenv("RIG_PROJECT_CONFIG_FILE")
 
-	var discovery []string
 	if projectConfigFile == "" {
-		discovery = make([]string, 2)
-		discovery[0] = "./outrigger.yml"
-		discovery[1] = "./.outrigger.yml"
-	} else {
-		discovery = make([]string, 1)
-		discovery[0] = projectConfigFile
+		projectConfigFile, _ = FindProjectConfigFilePath()
 	}
 
-	readyConfig := &ProjectConfig{}
-	for _, filePath := range discovery {
-		if config, err := NewProjectConfigFromFile(filePath); err == nil {
+	if projectConfigFile != "" {
```

Comment:
How did this PR sweep up the recursive config loading?

Reply:
I think I rebased at some point, but this is weird. Maybe I will rebase on develop again just for good measure